src/store/collections/TransactionsList.java

```diff
@@ -1,5 +1,5 @@
 package store.collections;
 
-public class TransactionsList {
-
-}
+public class TransactionsList{
+	
+}
```

Comment:
Not sure if he will nick us on this but for coding format there should be a space right here